src/components/tabell/brukernavn.tsx

```diff
@@ -22,15 +25,11 @@ const BrukerNavn = ({className, bruker, enhetId}: BrukerNavnProps) => {
 
     return (
         <div className={className}>
-            <Link
-                onClick={() => {
-                    setFraBrukerIUrl(bruker.fnr);
-                }}
-                href={getPersonUrl(bruker.fnr, '', enhetId)}
-                className={classnames('lenke lenke--frittstaende')}
-            >
-                <TekstKolonne skalVises={true} tekst={settSammenNavn(bruker)} />
-            </Link>
+            <AksjonKnappMedPopoverFeilmelding
+                aksjon={handterKlikk}
+                knappStil="brukernavn__knapp__tekst"
```

Comment:
Kunne man kanskje like godt bruke .juster-tekst-venstre her? da ser man "stilen" tydligere og slepper å lage en ny fil for css'en? :)